### ISSUE TITLE
Loss of room-scale tracking if tracking is lost

### ISSUE THREAD

**Issue by Alexees** [Microsoft/MixedRealityToolkit-Unity#1353]
## Expected Behavior
Whenever the tracking is lost and resumed, the camera should readjust position and resume room-scale tracking.

## Actual Behavior
The camera stays at the height where it lost the tracking and stays there, even though tracking was picked up again. From the looks of the BoundaryManager TrackingSpaceType.RoomScale is used only once in Awake for setup. I couldn't find anything related to detecting changes in tracking state.
P.S.: This is an issue detected by Microsoft themselves when they reviewed our app.

## Unity Editor Version
MRTP4

## Mixed Reality Toolkit Release Version
status from last Wednesday

**Comment by Abstiens** [comment]
This is also happening without the Mixed Reality Toolkit in a new Unity Project. Anyone knows a workaround?

**Comment by StephenHodgson** [comment]
So this is a unity bug?

**Comment by StephenHodgson** [comment]
Please add repro steps

**Comment by Abstiens** [comment]
If @Alexees and I are talking about the same problem, it is a Unity Issue imo

- Enter Playmode 
=> Positional Tracking is working
- Make the Immersive Headset loose positional tracking by holding hands on the cameras
- Release your hands again 
=> Positional  Tracking won't work until you leave and reenter playmode (restart the app)

**Comment by StephenHodgson** [comment]
I believe this is also related to #1324

**Comment by Abstiens** [comment]
Probably.
I also tried the new patch version that was released today (2017.2.0p2) but no luck.
I guess there is nothing you can do about it, right?

**Comment by StephenHodgson** [comment]
Probably best to keep to the MRTP builds

**Comment by Alexees** [comment]
@StephenHodgson if I'm not mistaken, the reason this doesn't work is because the UnityEngine.VR.WSA.WorldManager class is nowhere to be seen. 

`https://developer.microsoft.com/en-us/windows/mixed-reality/tracking_loss_in_unity`

**Comment by Abstiens** [comment]
fixed in MRTP 5 which is available now

**Comment by cre8ivepark** [comment]
Confirmed fix in MRTP5
https://forum.unity.com/threads/custom-build-2017-2-rc-mrtp-windows-mixed-reality-technical-preview.498253/

**Comment by keveleigh** [comment]
Looks like GitHub didn't automatically close these issues when merged into master. I'll close these manually and look into why.
